### PR TITLE
cvs: add page

### LINKED ISSUE #6373: page request: cvs
http://www.nongnu.org/cvs/

### PR DESCRIPTION
**Pull request by flyingcakes85**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

closes #6373

### CODE REVIEW

**Comment by pixelcmtd** [comment]
closes #6373

File: pages/common/cvs.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# cvs
+
+> Concurrent Versions System, a revision control system.
+> More information: <http://cvs.nongnu.org>.
+
+- Create repository (need to manually set CVSROOT to /path/to/repo in .bashrc):
+
+`cvs -d /path/to/repo init`
```
Comment (pixelcmtd) [inline pages/common/cvs.md:8]:
```suggestion
`cvs -d {{path/to/repo}} init`
```

File: pages/common/cvs.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Add project to repository. Navigate to project folder and run:
+
+`cvs import -m "message" projectname version vendor`
```
Comment (pixelcmtd) [inline pages/common/cvs.md:12]:
```suggestion
`cvs import -m "{{message}}" {{projectname}} {{version}} {{vendor}}`
```

File: pages/common/cvs.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Checkout a project:
+
+`cvs checkout projectname`
```
Comment (pixelcmtd) [inline pages/common/cvs.md:16]:
```suggestion
`cvs checkout {{project_name}}`
```

File: pages/common/cvs.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Check file diff/changes:
+
+`cvs diff file.txt`
```
Comment (pixelcmtd) [inline pages/common/cvs.md:20]:
```suggestion
`cvs diff {{path/to/file}}`
```

File: pages/common/cvs.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- Add a file:
+
+`cvs add file.txt`
```
Comment (pixelcmtd) [inline pages/common/cvs.md:24]:
```suggestion
`cvs add {{path/to/file}}`
```

File: pages/common/cvs.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- Commit a file:
+
+`cvs commit -m "commit message" file.txt`
```
Comment (pixelcmtd) [inline pages/common/cvs.md:28]:
```suggestion
`cvs commit -m "{{message}}" {{path/to/file}}`
```

**Review by pixelcmtd — CHANGES_REQUESTED** [review]

**Comment by flyingcakes85** [comment]
Thanks @pixelcmtd  for the review! I've made relevant changes.

File: pages/common/cvs.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+> Concurrent Versions System, a revision control system.
+> More information: <http://cvs.nongnu.org>.
+
+- Create repository (need to manually set CVSROOT to /path/to/repo in .bashrc):
+
+`cvs -d {{path/to/repo}} init`
+
+- Add project to repository. Navigate to project folder and run:
+
+`cvs import -m "{{message}}" {{projectname}} {{version}} {{vendor}}`
```
Comment (pixelcmtd) [inline pages/common/cvs.md:12]:
```suggestion
`cvs import -m "{{message}}" {{project_name}} {{version}} {{vendor}}`
```

File: pages/common/cvs.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
+# cvs
+
+> Concurrent Versions System, a revision control system.
+> More information: <http://cvs.nongnu.org>.
+
+- Create repository (need to manually set CVSROOT to /path/to/repo in .bashrc):
```
Comment (pixelcmtd) [inline pages/common/cvs.md:6]: As a non-CVS user, I'm really confused by this description. What is a `CVSROOT`? Why do I need to set it? And what does the configuration of a shell I don't use have to do with this? This really needs some clarification.

Comment (flyingcakes85) [reply pages/common/cvs.md:6]: @pixelcmtd 
I believe thats unclear wording on my part.
The way CVS works is that you have a CVSROOT environment variable that points to the repo path (which need not be on your local machine). The repo contains multiple projects (or modules), added via `cvs import` command.

This is different from git in the sense that you can have as many git repo on your system as you want. You switch to a different directory that has `.git` folder and further git commands will associate with that git repo. With CVS however, you define CVSROOT environment variable that points to the repo. Any cvs command irrespective of the working directory will execute with reference to path in this variable.

To be clear, the variable needs to be set _before_ any cvs command is run.

> And what does the configuration of a shell I don't use have to do with this?

I assumed the user are using bash, but yeah I get it, they could be using any shell. (I'm myself a zsh user). I suppose I could write it better as `Create repository (requires CVSROOT environment variable having path to repository)`. I'll make the changes if you say. I'm not the best at giving brief one liners.

PS.
I'm myself not a regular CVS user. Every project I need to see uses Git. I had gone through the basics of CVS some time back simply out of interest.

Comment (pixelcmtd) [reply pages/common/cvs.md:6]: Wow.

PS: I'd phrase it a bit differently like ``Create repository (requires the `CVSROOT` environment variable to be set externally)``.

Comment (flyingcakes85) [reply pages/common/cvs.md]: I've updated the command info :+1:

**Review by pixelcmtd — APPROVED** [review]

File: pages/common/cvs.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
+# cvs
+
+> Concurrent Versions System, a revision control system.
+> More information: <http://cvs.nongnu.org>.
+
+- Create repository (requires the `CVSROOT` environment variable to be set externally):
```
Comment (marchersimon) [inline pages/common/cvs.md:6]:
```suggestion
- Create a new repository (requires the `CVSROOT` environment variable to be set externally):
```

File: pages/common/cvs.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+`cvs -d {{path/to/repo}} init`
```
Comment (marchersimon) [inline pages/common/cvs.md:8]:
```suggestion
`cvs -d {{path/to/repository}} init`
```

File: pages/common/cvs.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+- Add project to repository. Navigate to project folder and run:
```
Comment (marchersimon) [inline pages/common/cvs.md:10]:
```suggestion
- Add a project to the repository:
```

File: pages/common/cvs.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+
+`cvs import -m "{{message}}" {{project_name}} {{version}} {{vendor}}`
+
+- Checkout a project:
+
+`cvs checkout {{project_name}}`
+
+- Check file diff/changes:
```
Comment (marchersimon) [inline pages/common/cvs.md:18]:
```suggestion
- Show changes made to files:
```

File: pages/common/cvs.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+
+- Add a file:
+
+`cvs add {{path/to/file}}`
+
+- Commit a file:
+
+`cvs commit -m "{{message}}" {{path/to/file}}`
+
+- Update working directory from repository:
```
Comment (marchersimon) [inline pages/common/cvs.md:30]: Is this correct?
```suggestion
- Update the working directory from the remote repository:
```

File: pages/common/cvs.md [hunk 14]
```diff
@@ -0,0 +1,32 @@
+# cvs
+
+> Concurrent Versions System, a revision control system.
+> More information: <http://cvs.nongnu.org>.
```
Comment (marchersimon) [inline pages/common/cvs.md:4]:
```suggestion
> More information: <https://cvs.nongnu.org>.
```

File: pages/common/cvs.md (hunk 13 above)
Comment (flyingcakes85) [reply pages/common/cvs.md:30]: Yes this will be right

**Review by marchersimon — APPROVED** [review]
Thanks @flyingcakes85

**Pull request merged**